resources/kds-team.ex-activecampaign/templates/Api.json

```diff
@@ -0,0 +1,14 @@
+  "api": {
```

Comment:
you don't need this "api", it actually makes the file invalid json.  Just start with `{`

Reply:
I think this file should look more like https://github.com/keboola/kbc-ui-templates/blob/633eed57d16bbdc89743dc1233522a8408c5c831/resources/keboola.ex-portadi/templates/api.json for example